amd_openvx_extensions/amd_rpp/source/Brightness.cpp

```diff
@@ -21,100 +21,102 @@ THE SOFTWARE.
 */
 
 #include "internal_publishKernels.h"
-#define NUM_OF_DIMS 5
 
-struct BrightnessLocalData
-{
+struct BrightnessLocalData {
     RPPCommonHandle handle;
     rppHandle_t rppHandle;
-    Rpp32u device_type;
+    Rpp32u deviceType;
     Rpp32u nbatchSize;
     RppPtr_t pSrc;
     RppPtr_t pDst;
     vx_float32 *alpha;
     vx_float32 *beta;
-    RpptDescPtr src_desc_ptr;
+    RpptDescPtr srcDescPtr;
     RpptDesc srcDesc;
     RpptDesc dstDesc;
-    RpptDescPtr dst_desc_ptr;
-    RpptROI *roi_tensor_ptr;
+    RpptDescPtr dstDescPtr;
+    void *roiTensorPtr;
+    RpptROI *roiPtr;
     RpptRoiType roiType;
-    Rpp32u layout;
-    size_t in_tensor_dims[NUM_OF_DIMS];
-    size_t out_tensor_dims[NUM_OF_DIMS];
-    vx_enum in_tensor_type;
-    vx_enum out_tensor_type;
+    Rpp32s inputLayout;
+    Rpp32s outputLayout;
+    size_t inputTensorDims[NUM_OF_DIMS];
+    size_t ouputTensorDims[NUM_OF_DIMS];
+    vx_enum inputTensorType;
+    vx_enum outputTensorType;
 #if ENABLE_HIP
     void *pSrc_dev;
```

Comment:
Can we change this to pSrcDev?

Reply:
This was suggested and approved by Rajy in previous PR, hence will retain the same